app/config/default-env.json

```
@@ -0,0 +1,19 @@
+{
+  "version": "1.14.4",
+  "minimumCoreVersion": "5.2.0",
+  "protocol": "https",
+  "host": "${OPENHIM_CORE_MEDIATOR_HOSTNAME}",
+  "hostPath": "",
+  "port": "${OPENHIM_MEDIATOR_API_PORT}",
+  "title": "Admin Console",
+  "footerTitle": "OpenHIM Administration Console",
+  "footerPoweredBy": "<a href='http://openhim.org/' target='_blank'>Powered by OpenHIM</a>",
+  "loginBanner": "",
+  "mediatorLastHeartbeatWarningSeconds": 60,
+  "mediatorLastHeartbeatDangerSeconds": 120,
+  "showLoginForm": true,
+  "ssoEnabled": ${KC_OPENHIM_SSO_ENABLED},
```

```suggestion
  "ssoEnabled": ${KC_OPENHIM_SSO_ENABLED},
```
@michaelloosen  in platform this value is within double quotes, as string it's evaluated to "true".